packages/packages/commonwell-sdk/src/models/organization.ts

```diff
@@ -1,7 +1,91 @@
 import { z } from "zod";
+import { linkSchema } from "./link";
 
 export const organizationSchema = z.object({
```

Comment:
OOC, have you tried getting an org that has more info attached to it? ie. some NPI numbers?

I think there could be more to this schema - would suggest adding some more info to an org in the portal, getting it, and see if we're missing some values